apps/journeys-admin/src/components/JourneyView/TemplateSettings/TemplateSettingsDialog/FeaturedCheckbox/FeaturedCheckbox.tsx

```diff
@@ -0,0 +1,46 @@
+import Checkbox from '@mui/material/Checkbox'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import FormGroup from '@mui/material/FormGroup'
+import Stack from '@mui/material/Stack'
+import { ChangeEvent, ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+
+interface FeaturedCheckboxProps {
+  loading: boolean
+  values: boolean
+  name: string
+  handleChange: (e: ChangeEvent<unknown>) => void
```

Comment:
```suggestion
  handleChange: (e: ChangeEvent<HTMLInputElement>) => void
```

I think MUI Checkbox onChange has type HTMLInputElement